src/storyTemplates/startingSchool.js

```diff
@@ -159,7 +159,12 @@ const startingSchool = {
     {
       // page 16
       pageTemplate: 'BackCover',
```

Comment:
it was failing to download because it was missing some URLs for the pictures and stuff. This is not the correct URL for the back cover, Just a copy of previous one. The weird thing is that it displayed back cover

Reply:
hmmm strange

Reply:
Hi Ako, that works but that image is on my cloudinary account. @Chloeh24  moved all the files over to a new account that we have/will give to Trina. She probably just missed this one when she was copying all the new links into the story the other day. You'll need to get the correct image link from her.